pkg/agent/controller/networkpolicy/audit_logging.go

```diff
@@ -188,25 +189,32 @@ func getNetworkPolicyInfo(pktIn *ofctrl.PacketIn, c *Controller, ob *logInfo) er
 
 	// Get disposition Allow or Drop.
 	match = getMatchRegField(matchers, openflow.APDispositionField)
-	info, err := getInfoInReg(match, openflow.APDispositionField.GetRange().ToNXRange())
+	disposition, err := getInfoInReg(match, openflow.APDispositionField.GetRange().ToNXRange())
 	if err != nil {
 		return fmt.Errorf("received error while unloading disposition from reg: %v", err)
 	}
-	ob.disposition = openflow.DispositionToString[info]
+	ob.disposition = openflow.DispositionToString[disposition]
 
 	// Set match to corresponding ingress/egress reg according to disposition.
-	match = getMatch(matchers, tableID, info)
+	match = getMatch(matchers, tableID, disposition)
 
 	if match != nil {
 		// Get NetworkPolicy full name and OF priority of the conjunction.
-		info, err = getInfoInReg(match, nil)
+		conjID, err := getInfoInReg(match, nil)
 		if err != nil {
 			return fmt.Errorf("received error while unloading conjunction id from reg: %v", err)
 		}
-		ob.npRef, ob.ofPriority = c.ofClient.GetPolicyInfoFromConjunction(info)
+		ob.npRef, ob.ofPriority, ob.ruleName = c.ofClient.GetPolicyInfoFromConjunction(conjID)
+		if ob.npRef == "" || ob.ofPriority == "" {
+			return fmt.Errorf("networkpolicy not found for conjunction id: %v", conjID)
+		}
+		// Placeholder for K8s NetworkPolicies without rule names.
+		if ob.ruleName == "" {
+			ob.ruleName = "<nil>"
```

Comment:
I would remove the parenthesis

Reply:
How about the existing [placeholder for port](https://github.com/antrea-io/antrea/blob/b0901475b19f0abf51ecf364061e6da64f5246a5/pkg/agent/controller/networkpolicy/audit_logging.go#L246) in the case of icmp, should we keep them consistent without the parenthesis?

Reply:
Removing parenthesis would lead to inability to differentiate between empty rule name and rule name "nil", though the latter is not common. I feel it's better to wrap such values with special characters. And we could perhaps unify all empty values to `<nil>`, including empty priority in K8s default rule, which uses -1 currently. Then it's easier to get the convention when reading the logs. It's just like kubectl's output, all empty values are printed as `<none>`:
```
NAME                 STATUS   ROLES           AGE   VERSION   INTERNAL-IP   EXTERNAL-IP   OS-IMAGE       KERNEL-VERSION      CONTAINER-RUNTIME
kind-control-plane   Ready    control-plane   24h   v1.24.0   172.18.0.2    <none>        Ubuntu 21.10   5.4.0-109-generic   containerd://1.6.4
kind-worker          Ready    <none>          24h   v1.24.0   172.18.0.4    <none>        Ubuntu 21.10   5.4.0-109-generic   containerd://1.6.4
kind-worker2         Ready    <none>          24h   v1.24.0   172.18.0.3    <none>        Ubuntu 21.10   5.4.0-109-generic   containerd://1.6.4
```

Reply:
Done, unified all empty values for audit logging to `<nil>`.